### PR TITLE
Copy activity/sequence and convert core question types to interactives

### PR DESCRIPTION
**Pull request by emcelroy**
- Update import and sequences API controllers.
- Add conversion script URL and template activity ID env vars.
- Add "convert" links in appropriate places.

[#174824504]

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -22,10 +22,14 @@ services:
       SHUTTERBUG_URI:
       # Enables logging to stdout instead of file.
       RAILS_STDOUT_LOGGING: "true"
-      #set to https in .env
+      # set to https in .env
```
Comment (emcelroy) [inline docker-compose.yml:25]: @scytacki I'm guessing I should have left this out of the commit. Is that right?

File: app/helpers/lightweight_activity_helper.rb [hunk 2]
```diff
@@ -67,4 +67,19 @@ def activity_player_url(activity)
...
+    else
+      params[:lara_resource] = "#{url_encode(api_v1_activity_url(activity.id))}.json"
+      params[:resource_name] = activity.name
+    end
+
+    url = ENV['CONVERSION_SCRIPT_URL']
+    uri = URI.parse url
+    uri.query = URI.encode_www_form URI.decode_www_form(uri.query || "").concat(params.to_a)
+    return uri.to_s
+  end
```
Comment (emcelroy) [inline app/helpers/lightweight_activity_helper.rb:84]: @scytacki Is this pretty much what you intended for adding the params to the conversion script URL?

Comment (scytacki) [reply app/helpers/lightweight_activity_helper.rb:84]: The way this is done mostly in the portal is like this:
https://github.com/concord-consortium/rigse/blob/60fdf4f894316382af1ab16925c3ef0eb1385ca1/rails/app/controllers/external_activities_controller.rb#L57-L62

That seems like the best way to go.

**Review by scytacki — CHANGES_REQUESTED** [review]
Just the one change needed with the parsing of the URL.

I also noticed this contains code that is duplicated in the other open PR. So once that one is merged it'd probably be best to merge master into this one and resolve any conflicts.

**Review by scytacki — APPROVED** [review]
This looks good to me.
It is still marked draft though, is there more changes you want to add to it?

It could use some more tests on `activity_player_conversion_url()`, but I don't think it is big deal if they are missing in this case.

**Review by scytacki — APPROVED** [review]
Looks good to me.

**Pull request merged**